src/OrbitQt/CallTreeWidget.cpp

```diff
@@ -351,6 +353,17 @@ static void CollapseRecursively(QTreeView* tree_view, const QModelIndex& index)
   }
 }
 
+void CallTreeWidget::OnAltKeyAndMousePressed(const QPoint& point) {
+  QModelIndex index = ui_->callTreeTreeView->indexAt(point);
+  if (!index.isValid()) return;
+
+  if (ui_->callTreeTreeView->isExpanded(index)) {
+    CollapseRecursively(ui_->callTreeTreeView, index);
+  } else {
+    ExpandRecursively(ui_->callTreeTreeView, index);
```

Comment:
After https://github.com/google/orbit/pull/3499 that I just merged, this requires wrapping with `DisableThreadOrFunctionColumnResizeOnRowExpanded();` (before) and `ReEnableThreadOrFunctionColumnResizeOnRowExpanded(); ResizeThreadOrFunctionColumnToShowVisibleDescendants(selected_index);` (after). Have a look at `CallTreeWidget::OnCustomContextMenuRequested`. Not ideal, but see that pull request for an explanation.